cpp/example_code/s3/get_put_object_acl.cpp

```diff
@@ -284,16 +284,24 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        const Aws::String bucket_name = "my-bucket";
-        const Aws::String object_key = "my-file.txt";
+        //TODO: Change to the name of an actual bucket in your account.
+        const Aws::String bucket_name = "DOC-EXAMPLE-BUCKET";
+        //TODO: Create a file called "my-file.txt" in the local folder where your executables are built to.
+        const Aws::String object_name = "my-file.txt";
+        //TODO: Set to the region of your profile.
+        const Aws::String region = "us-east-1";
 
-        // Set the ACL's owner information. 
+
+        //TODO: Set the ACL's owner information (if it is your bucket, then you want your canonical id). 
+        //See https://docs.aws.amazon.com/AmazonS3/latest/userguide/finding-canonical-user-id.html for more information.,
+        //or you can find it by running the executable run_get_acl.exe of this project. 
         const Aws::String owner_id = 
             "b380d412791d395dbcdc1fb1728b32a7cd07edae6467220ac4b7c0769EXAMPLE";
 
         // Set the ACL's grantee information.
         const Aws::String grantee_permission = "READ";
 
+        //TODO: Select which form of 'grantee' you want to specify, and update the corresponding data.
         // If the grantee is by canonical user, then either the user's ID or 
         // display name must be specified:
```

Comment:
display name? It doesn't seem to match the following const. The code seems obvious enough to not need an explanation.